CIP-????/README.md

```diff
@@ -0,0 +1,90 @@
+---
+CIP: ???
+Title: multi-arity uplc lambda node
+Category: Plutus
+Status: Proposed
+Authors:
+    - Michele Nuzzi <michele@harmoniclabs.tech>
+Implementors: []
+Discussions:
+    - https://github.com/cardano-foundation/CIPs/pull/1043
+Created: 2025-06-02
+License: CC-BY-4.0
+---
+
+<!-- Existing categories:
+
+- Meta     | For meta-CIPs which typically serves another category or group of categories.
+- Wallets  | For standardisation across wallets (hardware, full-node or light).
+- Tokens   | About tokens (fungible or non-fungible) and minting policies in general.
+- Metadata | For proposals around metadata (on-chain or off-chain).
+- Tools    | A broad category for ecosystem tools not falling into any other category.
+- Plutus   | Changes or additions to Plutus
+- Ledger   | For proposals regarding the Cardano ledger (including Reward Sharing Schemes)
+- Catalyst | For proposals affecting Project Catalyst / the Jörmungandr project
+
+-->
+
+## Abstract
+<!-- A short (\~200 word) description of the proposed solution and the technical issue being addressed. -->
+This CIP proposes a new node for an uplc tree.
```

Comment:
```suggestion
This CIP proposes a new node for an UPLC tree.
```
If there's any context where lowercase makes sense, please let reviewers & editors know... e.g. if it's a symbol (in which case it would be `uplc`).